assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convertors.py

```diff
@@ -319,3 +322,79 @@ def save_as_mlflow(self):
             conda_env=conda_env_file,
             code_path=code_path,
         )
+
+
+class LLaVAMLFlowConvertor(PyFuncMLFLowConvertor):
```

Comment:
nit: we can one parent class for task and child class for model family. May be useful for different model family for same task. But, yea, it may not be useful to you.
https://github.com/Azure/azureml-assets/blob/26c318856da68ef21e32bd97dcf8c961bcc02e80/assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convertors.py#L258

https://github.com/Azure/azureml-assets/blob/26c318856da68ef21e32bd97dcf8c961bcc02e80/assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convertors.py#L284

Reply:
Making pragmatic argument to refactor at a later date since the code works fine as is now.